### PR TITLE
endpoint for posting a new comment via article id - TASK 10

### PR DESCRIPTION
**Pull request by philjacks**
- endpoint added for posting a new comment via article id
- errors 404 & 400 tested
- new comment included in response object
- new error code added to psql handler

### CODE REVIEW

File: __tests__/endpoints.test.js [hunk 1]
```diff
@@ -289,3 +289,55 @@ describe("GET /api/articles/:article_id/comments", () => {
...
+    };
+    return request(app)
+      .post("/api/articles/beans/comments")
+      .send(newComment)
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Bad request");
+      });
+  });
+});
```
Comment (DCrawley94) [inline __tests__/endpoints.test.js:343]: These tests are good but here's a couple of other things to think about.

What if you post a comment with a user that doesn't exist in the DB?

What if we try to post without sending the correct data?

**Review by DCrawley94 — CHANGES_REQUESTED** [review]
Looking good Phil! Just a couple of things to think about (and test!)

File: __tests__/endpoints.test.js (hunk 1 above)
Comment (philjacks) [reply __tests__/endpoints.test.js:371]: sorted :)

**Review by DCrawley94 — APPROVED** [review]
Looking good Phil!

**Pull request merged**